### PR TITLE
Add text wrap to all Person fields to fix truncation

### PR DESCRIPTION
**Pull request by oeiyiping**
Previously, any field with an excessively long value will be truncated.
truncate

Resolves:
- #155
- #230

### CODE REVIEW

File: src/main/resources/view/PersonListCard.fxml [hunk 1]
```diff
@@ -16,24 +17,27 @@
     </columnConstraints>
     <VBox alignment="CENTER_LEFT" GridPane.columnIndex="0">
       <padding>
-        <Insets top="5" right="5" bottom="5" left="15" />
+        <Insets bottom="5" left="15" right="5" top="5" />
```
Comment (justgnohUG) [inline src/main/resources/view/PersonListCard.fxml:20]: May I ask if this was adjusted by yourself or?

File: src/main/resources/view/PersonListCard.fxml [hunk 2]
```diff
@@ -7,6 +7,7 @@
 <?import javafx.scene.layout.GridPane?>
 <?import javafx.scene.layout.HBox?>
 <?import javafx.scene.layout.Region?>
+<?import javafx.scene.layout.RowConstraints?>
 <?import javafx.scene.layout.VBox?>
 
 <HBox id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
```
Comment (justgnohUG) [inline src/main/resources/view/PersonListCard.fxml:13]: With reference to this, 
https://github.com/AY2021S2-CS2103T-T12-3/tp/pull/224

@tanboonji changed the JavaFX version to 11, might want to change to that.

**Comment by justgnohUG** [comment]
It doesn't seem to be working on my end. 

Do take a look at my comments.

image

File: src/main/resources/view/PersonListCard.fxml (hunk 1 above)
Comment (oeiyiping) [reply src/main/resources/view/PersonListCard.fxml:20]: All changes in this PR was changed via SceneBuilder.

**Comment by justgnohUG** [comment]
Other than the comments, works on my side.

image

File: src/main/resources/view/PersonListCard.fxml (hunk 2 above)
Comment (oeiyiping) [reply src/main/resources/view/PersonListCard.fxml]: fixed in cef34705b68df4750fd7acdc32d5bd7cb6bab442

**Comment by justgnohUG** [comment]
LGTM.

**Review by justgnohUG — APPROVED** [review]

**Pull request merged**